### PR TITLE
Add a converter for TLW Global K10-1220Z Smart LED Dimmer

### PR DESCRIPTION
**Pull request by busywait**
I bought a K10-1220Z Smart LED driver on eBay and it required this converter to make it work. 

- The K10-1220Z has a button concealed under a flap labelled "Prog" . Long pressing it causes the device to send a status message.
- Switch the device off and then on three times to enter pairing mode.
- The Stop and Finished effects do nothing

K10-1220Z [12V 15W Zigbee Smart LED Driver With 6-Port Micro Plug Connector (tlwglobal.com)](https://www.tlwglobal.com/product/smart-driver-12v-led-smart-driver-15w-with-6-port-micro-plug-connector/)

These two higher-power drivers from the same vendor are also enabled with the assumption that the model ID string matches the vendor website: 
K10-1230Z [12V 30W Zigbee Smart LED Driver With 6-Port Micro Plug Connector (tlwglobal.com)](https://www.tlwglobal.com/product/smart-driver-12v-led-smart-driver-30w-with-6-port-micro-plug-connector/)
K10-1250Z [12V 50W Zigbee Smart LED Driver With 6-Port Micro Plug Connector (tlwglobal.com)](https://www.tlwglobal.com/product/smart-driver-12v-led-smart-driver-50w-with-6-port-micro-plug-connector/#)

### CODE REVIEW

File: devices/tlwglobal.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    // Tested working with firmare 2.5.3_r58: dimming, on/off, and effects give no 
+    // errors (although the stop effect and the finish effect do nothing).
+    {
+        zigbeeModel: ['K10-1220Z'],
+        model: 'K10-1220Z',
+        vendor: 'TLW Global',
+        description: '12V LED smart driver 15W with 6-port micro plug connector',
+        extend: extend.light_onoff_brightness(),
+    },
+    // K10-1230Z and K10-1250Z untested, but assumed to be consistent with K10-1220W
```
Comment (Koenkk) [inline devices/tlwglobal.js:23]: Please uncomment the commented definitions, they look good to me.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2010

File: devices/tlwglobal.js (hunk 1 above)
Comment (busywait) [reply devices/tlwglobal.js:23]: > Please uncomment the commented definitions, they look good to me.

OK, I've uncommented the definitions for K10-1230Z and K10-1250Z. I guess someone will tell you if the IDs don't match :D

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**